Backend/Tournament_User/TournamentUserMVSC/server.js

```diff
@@ -0,0 +1,181 @@
+const express = require('express');
+const app = express();
+const mysql = require('mysql2');
+
+// Set up the MySQL connection
+const db = mysql.createConnection({
+    host: '202.166.134.202',
+    user: 'root',   // Replace with your DB username
+    password: 'UnZ7Sqx26eFpwfBsLXC8KE',   // Replace with your DB password
+    port: 32781,
+    database: 'TournamentMSVC' // Replace with your DB name
+});
+
+// Connect to MySQL
+db.connect((err) => {
+    if (err) {
+        console.error('Error connecting to the database:', err);
+        return;
+    }
+    console.log('Connected to the MySQL database');
+});
+
+// Middleware to parse incoming JSON
+app.use(express.json());
+
+// Sample route to get tournament history
+app.get('/tournaments', (req, res) => {
+    const sql = `CALL GetTournamentHistory();`;
+    
+    db.query(sql, (err, results) => {
+        if (err) {
+            res.status(500).send('Error retrieving data from database');
+        } else {
+            res.status(200).json(results);
+        }
+    });
+});
+
+//Route to get tournamentId 
+
+app.get('/tournaments/GetTournamentMatchups/:tournamentId',(req,res)=>{
```

Comment:
for your route naming convention, i think can refer to the README.md file
for route naming convention, typically it should just be nouns? so in this case it would be similar to GET /tournaments/:tournamentId